changelog.md

```diff
@@ -8,6 +8,8 @@
 
 2024-10-18 - 59c22c85b1256cc167408e93402f6fb85a3bb775 -Input/PasswordInput.vue - Updating password Input field
 
+2024-10-11 - 6d028db15ec82666c290476cc656467c1e95b9aa - LinkWithPill.vue, link-with-pill.spec.ts, ProfilePicture.vue, ProfilePic.vue, profile-pic.spec.ts, ListSkeleton.story.vue - add onclick event to prevent default behaviour of link and execute a function
+
```

Comment:
Best to update this with recent date because it might be difficult to track change